.github/workflows/validate.yml

```diff
@@ -0,0 +1,10 @@
+name: Validate renovate config
+
+on: pull_request
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: renovate-config-validator
+        uses: rinchsan/renovate-config-validator@v0.0.11
```

Comment:
I'm not sure if we have a hard rule, but I personally would definitely prefer if we used the same CI consistently. This is super simple, so I'd be happy to compromise. Not sure how strict we want to be here

Reply:
changed to circleci